src/main/java/org/openstreetmap/atlas/utilities/conversion/ProtoLocationConverter.java

```diff
@@ -0,0 +1,31 @@
+package org.openstreetmap.atlas.utilities.conversion;
+
+import org.openstreetmap.atlas.geography.Latitude;
+import org.openstreetmap.atlas.geography.Location;
+import org.openstreetmap.atlas.geography.Longitude;
+import org.openstreetmap.atlas.proto.ProtoLocation;
+
+/**
+ * Converts back and forth between ProtoLocation and Location
+ *
+ * @author lcram
+ */
+public class ProtoLocationConverter implements TwoWayConverter<ProtoLocation, Location>
```

Comment:
Nice-To-Have: Corresponding unit test for each converter. You're doing this in your Builder Test, but if someone needs to change this, would be good to have a unit test for each. If you feel this is too much, feel free to disregard.

Reply:
This is actually a good idea. Shouldn't be too hard. It'll be in a future commit.

Reply:
NOTE: Would it be better to move this class to org.openstreetmap.atlas.geography.converters?

Reply:
Yep - see my original comment about a new package for these! I'd suggest `org.openstreetmap.atlas.geography.converters.proto`

Reply:
On it.